pandas/io/parsers.py

```diff
@@ -1065,6 +1065,38 @@ def _evaluate_usecols(usecols, names):
     return usecols
 
 
+def _validate_usecols(usecols, names):
+    """
+    Validates that all usecols are present in a given
+    list of names. If not, raise a ValueError that
+    shows what usecols are missing.
+
+    Parameters
+    ----------
+    usecols : iterable of usecols
+        The columns to validate are present in names.
+    names : iterable of names
+        The column names to check against.
+
+    Returns
+    -------
+    usecols : iterable of usecols
+        The `usecols` parameter if the validation succeeds.
+
+    Raises
+    ------
+    ValueError : Detailing which usecols are missing, if any.
```

Comment:
I would word this slightly to say "Columns were missing.  Error message will list them."

Reply:
Done!